src/components/esn-app-grid/esn-app-grid.tsx

```diff
@@ -0,0 +1,80 @@
+import { Component, Prop, ComponentInterface, State, Host, h } from '@stencil/core';
+import fetchSvgIcon from '../../utils/fetch-icon';
+import { InternalItem } from './esn-app-grid.types';
+import { AppGridTogglerIcon } from './app-grid-icon/AppGridIcon';
+
+@Component({
+  tag: 'esn-app-grid',
+  styleUrl: 'esn-app-grid.scss',
+  shadow: true
+})
+export class EsnAppGrid implements ComponentInterface {
+  private popover: HTMLElement;
+
+  @State() internalItems: InternalItem[] = [];
```

Comment:
Badly named, this is not an item, but an application. And do we need to name `internal` things `internalSomething`?

Reply:
Right, since in the issue Michael named it `items`, my mind kinda got stuck around it :)).

Reply:
Fixed.